src/test/scss/scssCompletion.test.ts

```diff
@@ -88,6 +88,36 @@ suite('SCSS - Completions', () => {
 				{ label: '.foo' }
 			]
 		});
+		testCompletionFor('@', {
+			items: [
+				{ label: '@extend' },
+				{ label: '@at-root' },
+				{ label: '@debug' },
+				{ label: '@warn' },
+				{ label: '@error' },
+				{ label: '@if' },
+				{ label: '@for', insertTextFormat: InsertTextFormat.Snippet },
+				{ label: '@each', insertTextFormat: InsertTextFormat.Snippet },
+				{ label: '@while', insertTextFormat: InsertTextFormat.Snippet },
+				{ label: '@mixin', insertTextFormat: InsertTextFormat.Snippet },
+				{ label: '@include' }
+			]
+		});
+		testCompletionFor(`@for $i from 1 through 3 { .item-#{$i} { width: 2em * $i; } } @|`, {
```

Comment:
This fixes an undocumented existing issue: suppose you have 
```scss
@for $i from 1 through 3 {
  .item-#{$i} { width: 2em * $i; }
}

@|
```

Won't trigger any top level completion.